src/pages/Login.tsx

```diff
@@ -1,14 +1,45 @@
-import React from "react";
-import { withRouter } from "react-router-dom";
+import React, { useState, useEffect } from "react";
+import { RouteComponentProps, withRouter } from "react-router-dom";
+import md5 from "md5";
+import { useCookies } from "react-cookie";
 
 import { Button, Input, Typography } from "antd";
 
+import { API_DOMAIN } from "../config";
 import Header from "../components/Header";
 import "../styles/Login.css";
 
-const Login: React.FC = (props) => {
-  const onConfirmClick = () => {
-    // TODO: Write code here to try logging in (if successful - dashboard, if error then display error)
+const Login: React.FC<RouteComponentProps> = (props) => {
+  const [ username, setUsername ] = useState("");
+  const [ password, setPassword ] = useState("");
+  const [ cookies, setCookie ] = useCookies(['accessToken']);
+  const [ error, setError ] = useState(false);
+
+  useEffect(() => {
+    if (cookies['accessToken']) {
+      props.history.push("/Dashboard");
+    }
+  }, [cookies]);
+
+  const onConfirmClick = async (): Promise<void> => {
+    const hashedPassword = md5(password);
+    const response = await fetch(`${API_DOMAIN}users/authenticate/${username}`, {
```

Comment:
If there's no input in the username or the correct username is given but the wrong password is given the statement "Username and/or password is incorrect" doesn't pop up. I think it might be a good idea to also have the error statement pop up if there is no input in username and password

Reply:
@TCHE614 thanks for that, it seems I missed some scenarios for the login stuff. I've made the changes so it should work now!